### ISSUE TITLE
Name Property is missing for Button in button control.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#554]
**Describe the bug**

Button needs automation name set on it.

**Steps to reproduce the bug**

1. Turn on Narrator . (Keyboard Shortcut: Ctrl+Win+Enter).
2. Launch XAML Controls Gallery.
3. Navigate to Button in all controls page->activate it.
4. Navigate to Button under "Accent style applied to button" and observe narrator reading.

**Expected behavior**

Narrator should read it as Button button. i.e. one for name and other for control type. (as same as it is reading for all other buttons present on the pane e.g: For Reveal style button it is reading "Reveal style button, Button".

**Actual behavior**

Narrator is only reading Button which means either name or control type is missing. 

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/92955731-8a9b9300-f41a-11ea-918d-d56ec89a66b0.png)

**Version Info**

Gallery version (found on Settings page): 

1.2.15.0

**Additional context**

Copied old [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/26330442). Have not verified the problem still repros.
